Makefile

```diff
@@ -444,6 +444,15 @@ test-data: start-docker ## Add test data to the local instance.
 	@echo Login with a regular account username=user-1 password=user-1
 	@echo ========================================================================
 
+cypress-test-data: start-docker ## Add test data to the local instance.
+	$(GO) run $(GOFLAGS) -ldflags '$(LDFLAGS)' $(PLATFORM_FILES) config set TeamSettings.MaxUsersPerTeam 100
+	$(GO) run $(GOFLAGS) -ldflags '$(LDFLAGS)' $(PLATFORM_FILES) sampledata -w 4 -u 60 
+	@echo You may need to restart the Mattermost server before using the following
+	@echo ========================================================================
+	@echo Login with a system admin account username=sysadmin password=sysadmin
+	@echo Login with a regular account username=user-1 password=user-1
+	@echo ========================================================================
```

Comment:
Looks good though it feels redundant to me. 0/5 but what if we just modify the existing `test-data`?  I don't see any issue of doing so, probably longer time to generate test data depending on local machine (but actually when I tried, there seemed to be no difference at all).

Reply:
Agree with @saturninoabril -- unless there's a compelling reason to deviate here, wonder if we can reduce the `Makefile` targets.

Reply:
Works for me, I'll throw it in the primary test data command.